### ISSUE TITLE
wavetable synth

### ISSUE THREAD

**Issue by autumncheney** [stargateaudio/stargate#32]
**Briefly describe your usage of Stargate**
i just started using stargate, and i have a few feature ideas that could make this a great daw to use while also abiding under its own principles

**Describe what you want to do**
i would like to use a wavetable synth in stargate. wavetable synths are rare among libre synths, and adding one would make this daw stand out and make it even more powerful

After the high-level overview statement, you may provide additional detail and
context below.

**Does your proposal align with the core project design principles?**
i believe it does. if it doesn't, that's fine

**OPTIONAL: Describe one or more ways you propose we can accomplish this**

**Additional context**

**Comment by stargateaudio** [comment]
FM1 is a wavetable synthesizer, with the ability to do FM synthesis, and the ability to create wavetables using additive synthesis.  Are you looking for another type of feature, such as loading wavetables from audio files?

**Comment by stargateaudio** [comment]
Specifically, FM1 has about 17 pre-defined wavetables, and 3 user defined wavetables that are created using additive synthesis.

**Comment by stargateaudio** [comment]
Also, the built-in sampler can turn samples into wavetables using the "Tune" feature to set a loop length to musical time.

**Comment by autumncheney** [comment]
yes, a synth plugin where you can load wavetable (audio) files and scroll through the waveforms would be nice

**Comment by stargateaudio** [comment]
> a synth plugin where you can load wavetable (audio) files and scroll through the waveforms

I'm not sure I understand what `scroll through the waveforms` means.  Is there an existing plugin where I can see this functionality?

**Comment by autumncheney** [comment]
try surge or vitalium, they're both libre wavetable synths. for surge, adjust the "morph" slider, and for vitalium, adjust the slider next to the waveform display
